k8s/agent+helper/templates/_common.tpl

```diff
@@ -36,3 +36,15 @@ true
 {{- end -}}
 {{- end -}}
 {{- end -}}
+
+
+{{/*
```

Comment:
deprecated. please remove if there is no use in lber.

Reply:
removed